.github/workflows/dev_cd.yml

```diff
@@ -54,23 +54,17 @@ jobs:
   #   steps:
   #     - uses: actions/checkout@v2
   #       name: Check out code
-  #     - uses: docker/setup-qemu-action@v1
-  #       name: Set up QEMU
-  #     - uses: docker/setup-buildx-action@v1
-  #       name: Set up Docker Buildx
-  #     - uses: docker/login-action@v1
-  #       name: Login to DockerHub
+  #     - uses: mr-smithers-excellent/docker-build-push@v5
```

Comment:
Dumb question, but why did we change the library here?

Reply:
There is apparently an official version for this directly from Github. The one I was using before was fine but from a private person. Maybe we trust Github more, but really no special reason.